tests/helixprep.proj

```diff
@@ -0,0 +1,152 @@
+<Project ToolsVersion="12.0" DefaultTargets="ArchiveAll" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="ZipFileCreateFromDirectory" AssemblyFile="$(ToolsDir)\net45\Microsoft.DotNet.Build.Tasks.dll"/>
+  
+  <Import Project="dir.props" />
+  <Import Project="..\dir.props" />
+
+  <PropertyGroup>
+    <DiscoveryDirectory>$(TestWorkingDir)</DiscoveryDirectory>
+    <CoreRootDir Condition="'$(RuntimeID)' == '' ">$(CORE_ROOT)</CoreRootDir>
+    <CoreRootDir Condition="'$(RuntimeID)' != '' ">$(CORE_OVERLAY)</CoreRootDir>
+    <CoreRootName Condition="'$(RuntimeID)' == '' ">Core_Root_Windows_NT-$(__BuildArch)</CoreRootName>
+    <CoreRootName Condition="'$(RuntimeID)' != '' ">Core_Root_$(RuntimeID)</CoreRootName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <TestCmds Include="$(DiscoveryDirectory)\**\*.cmd" ></TestCmds>
+    <XunitDlls Include="$(DiscoveryDirectory)\*\*.XUnitWrapper.dll" ></XunitDlls>
+  </ItemGroup>
+
+  <!-- Build the platform-specific executable to run an individual xunit wrapper -->
+
+  <Target Name="GenerateWrapperExecutables"
```

Comment:
1. Good call, I used the wrong word. Will fix.
2. The child targets that are called (GenerateWrapperCmd, GenerateWrapperSh) have conditionals so they only run when being executed on the right OS, so we only build 1 per build.